index.js

```diff
@@ -39,11 +38,13 @@ if (process.env.hasOwnProperty('GCLOUD_DEBUG_REPO_APP_PATH')) {
 
 var log = logger.create(config.logLevel, '@google/cloud-debug');
 
-if (semver.satisfies(process.version, '5.2')) {
-  log.error('Debug is not supported on Node v5.2');
-  return;
+if (semver.satisfies(process.version, '5.2 || <0.12')) {
+  log.error('Debug is not supported on Node v5.2 or <v0.12');
+  process.exit(1);
```

Comment:
I am not so sure about the `process.exit`. It is quite possible for users to have dev environments different from the version of Node they would get on deployment. We are converting a lower severity problem ('my debug agent isn't starting') to a very high severity problem ('my app doesn't deploy').